Feat 직업 검사 결과 페이지 최상단에 차트 라벨 사용법 문구 추가

sejinbaek opened the pull request:
## 📝 변경 사항

> 커밋 단위로 명시적으로 짧게 작성합니다.

1. 직업 검사 결과 페이지 최상단에 차트 라벨을 사용법을 알리는 문구를 추가하였습니다.
2. 문구 오른쪽에 배치할 아이콘을 추가하였습니다.
3. 폰트 사이즈를 xl -> lg로 변경하였습니다.

## 🔍 변경 사항 세부 설명

> 세부 설명을 작성합니다.

- UX 개선을 위해 차트 라벨 클릭 시 카드가 나오는 설명 문구를 추가하였습니다.
- 통일성 있는 스타일을 위해 theme.js를 적용하였습니다.
- 차트와 문구 사이 간격을 줄여 한 화면에서 문구와 차트를 볼 수 있도록 차트와 카드 영역의 margin-top을 -70px로 설정하였습니다.
- 폰트 사이즈 theme의 xl(32px)로 적용했다가 너무 커서 lg로 줄였습니다.

## 🕵️‍♀️ 요청사항

> 팀원들에게 테스트나 리뷰를 요청합니다.

- 직업 검사 결과 페이지 최상단에 문구가 제대로 보이는지 확인 부탁드립니다.

## 📷 스크린샷 (선택)

> UI 변경이 있는 경우 스크린샷이나 GIF를 첨부합니다.

### 폰트 사이즈: xl
![image](https://github.com/user-attachments/assets/395a5abb-fa54-4c55-b7d6-40ae04e498b3)

### 폰트 사이즈: lg
![image](https://github.com/user-attachments/assets/3758a3d2-459b-40d2-8f77-89410de77b67)


## ✅ 작성자 체크리스트

- [x] Self-review: commit 이나 오타 없이 코드가 스스로 검토됨
- [x] 로컬에서 모든 기능이 정상 작동함(버그수정 /기능에 대한 테스트)
- [x] 린터 및 포맷터로 코드 정리됨

sejinbaek committed:
Chore: 차트 라벨 사용법 문구 옆 배치 아이콘 파일 추가

sejinbaek committed:
Feat: 직업 검사 결과 페이지 최상단 차트 라벨 사용법 문구 추가

- 사용자가 쉽게 접근할 수 있도록 차트 라벨 사용 방법 문구를 추가하였습니다.
- 통일성 있는 스타일을 위해 theme.js을 적용하였습니다.

yeom-kenco reviewed (approved):
정말 빠르시군요... 저희 이제 자러가죠 ㅎㅎㅎ

H-JuKyung reviewed (approved):
빠른 피드백 반영 감사합니다👍🏻🙌
늦은 시간까지 고생 많으셨어요ㅜㅜ

sejinbaek committed:
Feat: LayoutTitle 섹션 반응형 개선 (clamp, margin, media query 적용)

- h1 폰트 크기를 clamp로 반응형 대응
- image-wrapper의 크기 clamp 적용 및 margin-left, margin-top로 위치 보정
- 768px, 485px 미디어쿼리로 모바일 대응
- translateX 대신 media query에서 h1 위치 조정으로 자연스러운 반응형 배치

sejinbaek committed:
Style: 폰트 사이즈 xl -> lg로 변경

yeom-kenco committed:
Merge branch 'main' into Feat-차트-설명-문구-추가

hyonun321 commented:
수고하셨습니다 딱 어울리는 UI네요~
